src/algebra/big_operators.lean

```diff
@@ -1466,3 +1469,4 @@ begin
 end
 
 end with_top
+#lint
```

Comment:
Please remove the leftover lint.

Reply:
Ha! We should have linter for catching this (-;

Reply:
This must have generated output. Why is CI not complaining? All the checkmarks are green.